articles/flows/guides/device-auth/includes/user-authorization.md

```diff
@@ -0,0 +1,24 @@
+## User Authorization
+
+The user will either scan the QR code, or else will open the activation page and enter the user code:
+
+![Enter User Code](/media/articles/flows/guides/device-auth/enter-user-code.png)
```

Comment:
Just a suggestion, maybe we should use the tenant from the playground or any other tenant instead of the one from sandrino

Reply:
Done.